it/config/DocumentationControllerISpec.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package config
+
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.ws.WSResponse
+import support.IntegrationBaseSpec
+
+class DocumentationControllerISpec extends IntegrationBaseSpec {
+
+  val apiDefinitionJson: JsValue = Json.parse(
+    """
+    |{
+    |   "scopes":[
+    |      {
+    |         "key":"read:self-assessment",
+    |         "name":"View your Self Assessment information",
+    |         "description":"Allow read access to self assessment data"
+    |      },
+    |      {
+    |         "key":"write:self-assessment",
+    |         "name":"Change your Self Assessment information",
+    |         "description":"Allow write access to self assessment data"
+    |      }
+    |   ],
+    |   "api":{
+    |      "name":"#mtd-api# (MTD)",
+    |      "description":"#desc#",
+    |      "context":"mtd/template",
```

Comment:
Need to change this to reflect the changes made for the documentation controller